code/game/turfs/walls/wall_types.dm

```diff
@@ -273,11 +273,24 @@
 //	icon_state = "title_holiday"
 	layer = FLY_LAYER
 	pixel_x = -64
+	var/total_titles = 35 //<--- just change this if adding images etc
```

Comment:
This is a really bad solution, use https://www.byond.com/docs/ref/#/proc/icon_states to get the number of icon states in the dmi file instead.